### PR TITLE
Updating version notice, and fixing version detection logic

### PR DESCRIPTION
**Pull request by kiiadi**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description

Adding deprecation to the title of the message for a stronger call to action.

Screen Shot 2019-10-09 at 12 58 09 PM
Screen Shot 2019-10-09 at 1 06 39 PM

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/notification/JetBrainsMinimumVersionChange.kt [hunk 1]
```diff
@@ -29,7 +30,7 @@ class JetBrainsMinimumVersionChange : NoticeType {
     override fun isNotificationRequired(): Boolean {
         val appInfo = ApplicationInfo.getInstance()
         val majorVersion = appInfo.majorVersion.toIntOrNull()
-        val minorVersion = appInfo.minorVersion.toIntOrNull()
+        val minorVersion = appInfo.minorVersion.toFloatOrNull()
```
Comment (justinmk3) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/notification/JetBrainsMinimumVersionChange.kt]: Strange. Was that a JB/legacy/historical choice , or why was a float used?

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/notification/JetBrainsMinimumVersionChange.kt]: It's actually a string "2.2" which obviously didn't parse as an `Int`

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/notification/JetBrainsMinimumVersionChange.kt]: the `toFloatOrNull` is a kotlin built-in extension function on `String`

Comment (justinmk3) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/notification/JetBrainsMinimumVersionChange.kt]: In a version string x.y.z., "y" is the minor version and "z" is the "patch", and both could be integers then.

**Pull request merged**